selenium/build.sh

```diff
@@ -85,13 +87,14 @@ download_yandexdriver() {
 }
 
 download_selenoid() {
-    local tag=""
+    local download_url=""
+
     if [ "$1" == "latest" ]; then
-        tag="$1"
+        download_url=$(wget -qO- "https://api.github.com/repos/aerokube/selenoid/releases/$tag" | jq -r '.assets[].browser_download_url | select(contains("linux_amd64"))')
```

Comment:
@M4nouel This command is broken. You should use $1 instead of $tag. Apart from that everything else in the PR looks good, but please test all combinations before you're done. If you like, you can also remove empty lines inside those functions. cc: @vania-pooh

Reply:
Well received, sorry for the oblivion, I will be more vigilant